docs/core-features/table_lookups.md

```diff
@@ -1,6 +1,13 @@
 # Table lookup
 
-In TFHE, there exists mainly two operations: the linear operations (additions, subtractions, multiplications by integers) and the rest. And the rest is done with table lookups (TLUs), which means that a lot of things are done with TLU. In this document, we explain briefly, from a user point of view, how TLU can be used. In [the TLU advanced documentation](table_lookups_advanced.md), we enter a bit more into the details.
+{% hint style="success" %}
+**Zama 5-Question Developer Survey**
```

Comment:
this survey ad is twice in the same .md, it's too much to me, CC @yuxizama

Reply:
@yuxizama wdyt?

Reply:
We just need one survey banner here. Could you delete the first one, please? Thank you!